packages/paste-core/components/tabs/src/Tab.tsx

```diff
@@ -111,21 +99,20 @@ const Tab = React.forwardRef<HTMLDivElement, TabProps>(({children, ...tabProps},
   return (
     <TabPrimitive {...(tab as any)} {...tabProps} ref={ref}>
       {(props: TabProps) => {
-        const textStyles = getTabTextStyles(props);
         return (
           <Box
             {...safelySpreadBoxProps(props)}
             {...boxStyles}
-            position="relative"
             cursor={props['aria-disabled'] ? 'not-allowed' : 'pointer'}
-            transition="border-color 100ms ease"
-            textOverflow={tab.orientation !== 'vertical' ? 'ellipsis' : undefined}
             overflow={tab.orientation !== 'vertical' ? 'hidden' : undefined}
+            position="relative"
+            textOverflow={tab.orientation !== 'vertical' ? 'ellipsis' : undefined}
+            transition="border-color 100ms ease"
             whiteSpace={tab.orientation !== 'vertical' ? 'nowrap' : undefined}
           >
             <Text
```

Comment:
Should we remove this `Text` component and put the styles on the wrapping `Box`?

Reply:
Good call. Those styles can all go on `Box`.